ivadomed/training.py

```diff
@@ -283,23 +316,32 @@ def train(model_params, dataset_train, dataset_val, training_params, log_directo
     if model_params["name"] == "FiLMedUnet" and debugging:
         save_film_params(gammas_dict, betas_dict, contrast_list, model_params["depth"], log_directory)
 
-    try:
-        # Convert best model to ONNX and save it in model directory
-        best_model_path = os.path.join(log_directory, model_params["folder_name"], model_params["folder_name"] + ".onnx")
-        imed_utils.save_onnx_model(torch.load(model_path), input_samples, best_model_path)
-    except:
-        # Save best model in model directory
-        best_model_path = os.path.join(log_directory, model_params["folder_name"], model_params["folder_name"] + ".pt")
-        torch.save(model, best_model_path)
-        logger.warning("Failed to save the model as '.onnx', saved it as '.pt': {}".format(best_model_path))
+    # Save best model in log directory
+    if os.path.isfile(resume_path):
+        state = torch.load(resume_path)
+        model_path = os.path.join(log_directory, "best_model.pt")
+        torch.save(state["model"], model_path)
```

Comment:
state['model'] doesn't exist here (we don't have any key named model, we only have the state dict)

Reply:
```suggestion
        torch.save(model.load_state_dict(checkpoint['state_dict']), model_path)
```